TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Project Objective(`unique_id`)
+=======
+
+Procedure
+-----------
+* A
+* B
+* C
+
+ ```<Any Code If Required can be added like this.>```
```

Comment:
@iamads Code should more or less be added as source and any code that has to be written, links can be provided for that(github links)

Also Points A, B, C should have External Links to projects